odk1-src/form-logic.rst

```diff
@@ -0,0 +1,456 @@
+
+Statically defined repeats
+----------------------------
+
+Use the :th:`repeat_count` columns
+to define the number of times a group will repeat.
+
+
+.. rubric:: XLSForm
+
+.. csv-table:: survey
+  :header: type, name, label, repeat_count
+
+  begin_repeat, my_repeat, Repeat group label, 3
+  note, repeated_note, These questions will be repeated as an entire group.
+  text, name, What is your name?
+  text, quest, What is your quest?
+  text, fave_color, What is your favorite color?
+  end_repeat, , 
+ 
+.. _dynamically-defined-repeats:
+ 
+Dynamically defined repeats
+""""""""""""""""""""""""""""
+
+The :th:`repeat_count` column can reference
+:ref:`previous responses <variables>` and :ref:`calculations <calculations>`.
+
+.. rubric:: XLSForm
+
+.. csv-table:: survey
+  :header: type, name, label, repeat_count
+  
+  integer, number_of_children, How many children do you have?
+  begin_repeat, child_questions, Questions about child, ${number_of_children}
+  text, child_name, Child's name,
+  integer, child_age, Child's age,
+  end_repeat, , , 
+
+  
+.. _cascading-selects:
+  
+Filtering options based on previous responses
+===============================================
+
+To limit the options in a select question
+based on the answer to a previous question,
+use a :th:`choice_filter` row in the **survey** sheet,
+and filter key columns in the **choices** sheet.
+
+For example,
+you might ask the user to select a state first,
+and then only display cities within that state.
+This is called a `cascading select`_,
+and can be extended to any depth.
+`This example form`__ shows a three-tiered cascade:
+state, county, city.
+
+.. _cascading select: http://xlsform.org/#cascade
+
+__ https://docs.google.com/spreadsheets/d/1CCjRRHCyJXaSEBHPjMWrGotnORR4BI49PoON6qK01BE/edit#gid=0
+
+.. rubric:: XLSForm
+
+.. csv-table:: survey
+  :header: type, name, label, choice_filter
+  
+  select_one job_categories, job_category, Job category
+  select_one job_titles, job_title, Job title, job_category=${job_category} 
+
+.. csv-table:: choices
+  :header: list_name, name, label, job_category
+  
+  job_categories, finance, Finance,
+  job_categories, hr, Human Resources,
+  job_categories, admin, Administration/Office,
+  job_categories, marketing, Marketing,
+  job_titles, ar, Accounts Receivable, finance
+  job_titles, ap, Account Payable, finance
+  job_titles, bk, Bookkeeping, finance
+  job_titles, pay, Payroll, finance
+  job_titles, recruiting, Recruiting, hr
+  job_titles, training, Training, hr
+  job_titles, retention, Retention, hr
+  job_titles, asst, Office Assistant, admin
+  job_titles, mngr, Office Manager, admin
+  job_titles, scheduler, Scheduler, admin
+  job_titles, reception, Receptionist, admin
+  job_titles, creative_dir, Creative Director, marketing
+  job_titles, print_design, Print Designer, marketing
+  job_titles, ad_buyer, Ad Buyer, marketing
+  job_titles, copywriter, Copywriter, marketing
+
+.. _launch-apps-from-collect:
```

Comment:
Feels very out of place. I would move this back. And It's probably sufficient to drop the "Forms" from the title.